### PR TITLE
New Error Page (no chinese man)

### PR DESCRIPTION
**Pull request by dajinchu**

### CODE REVIEW

**Review by NEUDitao — APPROVED** [review]
https://www.youtube.com/watch?v=qL34_Oi5RH8

File: packages/app/components/common/FatalError.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Result from "antd/lib/result";
+import { ReactElement } from "react";
+
+export default function FatalError(): ReactElement {
+  return (
+    <Result
+      status="500"
+      title="We hit an unexpected error."
+      subTitle="Sorry about that! A report has automatically been filed. Try refreshing the page."
```
Comment (liustanley) [inline packages/app/components/common/FatalError.tsx:9]: where the report at doe 🤔

**Review by liustanley — APPROVED** [review]
tis a sad day

File: packages/app/components/common/FatalError.tsx (hunk 1 above)
Comment (dajinchu) [reply packages/app/components/common/FatalError.tsx:9]: we lyin

Comment (dajinchu) [reply packages/app/components/common/FatalError.tsx:9]: i think we would log it in Elastic APM

**Pull request merged**